add vector search operation and update integrations (#18273)
pingcap/docs: tidb-cloud/data-service-manage-endpoint.md

    2. Select at least one HTTP operation (such as `GET (Retrieve)`, `POST (Create)`, and `PUT (Update)`) for the endpoint to be generated.

        For each operation you select, TiDB Cloud Data Service will generate a corresponding endpoint. If you select a batch operation (such as `POST (Batch Create)`), the generated endpoint lets you operate on multiple rows in a single request.

>         If your table contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):

Reviewer: ```suggestion
        If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):
```

>         If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can click the **Vector Search Operations** toggle to generate a `POST (Vector Similarity Search)` endpoint, which automatically calculates vector distances based on the selected distance function. You can select from the following [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md):

Reviewer 2: ```suggestion
        If the table you selected contains [vector data types](/tidb-cloud/vector-search-data-types.md), you can enable the **Vector Search Operations** option and select a vector distance function to generate a vector search endpoint that automatically calculates vector distances based on your selected distance function. The supported [vector distance functions](/tidb-cloud/vector-search-functions-and-operators.md) include the following:
```

    - Endpoint name: the generated endpoint name is in the `/<name of the selected table>` format, and the request method (such as `GET`, `POST`, and `PUT`) is displayed before the endpoint name. For example, if the selected table name is `sample_table` and the selected operation is `POST (Create)`, the generated endpoint is displayed as `POST /sample_table`.

        - If a batch operation is selected, TiDB Cloud Data Service appends `/bulk` to the name of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Batch Create)`, the generated endpoint is displayed as `POST /sample_table/bulk`.

>         - If `POST (Vector Similarity Search)` is selected, TiDB Cloud Data Service appends `/vector_search` to the name  of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Vector Similarity Search)`, the generated endpoint is displayed as `POST /sample_table/vector_search`.

Reviewer: ```suggestion
        - If `POST (Vector Similarity Search)` is selected, TiDB Cloud Data Service appends `/vector_search` to the name of the generated endpoint. For example, if the selected table name is `/sample_table` and the selected operation is `POST (Vector Similarity Search)`, the generated endpoint is displayed as `POST /sample_table/vector_search`.
```
